src/helpers/chartHelpers.tsx

```diff
@@ -78,7 +78,7 @@ export function createBarChartDefs(layoutContext: LayoutContext, gradientKey: st
             </linearGradient>
         )}
         {options?.thresholds?.map((threshold, index) =>
-            <linearGradient key={`lg_thresh_${threshold}`} id={gradientKey + "_threshold" + index} x1="0" y1="0" x2="0" y2="1">
+            <linearGradient key={`lg_thresh_${index}`} id={gradientKey + "_threshold" + index} x1="0" y1="0" x2="0" y2="1">
```

Comment:
Is this change necessary?

Reply:
One of the charts was giving a react console warning about there being duplicate keys. Threshold is not necessarily a unique key value, so index seemed more appropriate.